### PR TITLE
DE Update 1.0

### PR DESCRIPTION
**Pull request by zRxnx**

### CODE REVIEW

File: language/de.json [hunk 1]
```diff
@@ -154,14 +154,14 @@
...
-    "adminstartedresource": "**%s** hat die Resource **%s** gestartet",
-    "adminstoppedresource": "**%s** hat die Resource **%s** gestoppt",
+    "adminstartedresource": "**%s** hat die Ressource **%s** gestartet",
+    "adminstoppedresource": "**%s** hat die Ressource **%s** gestoppt",
 
     "convarname": "Convar Name",
     "convarvalue": "Convar Inhalt",
 
     "cleanarea": "Gebiete aufräumen",
-    "cleanareaguide": "Entfernt alle Entities in der nähe.",
```
Comment (Blumlaut) [inline language/de.json:164]: Entities is what the V Engine calls these, there is no direct translation as "Objekte" can also be Props".

File: language/de.json [hunk 2]
```diff
@@ -154,14 +154,14 @@
     "setconvar": "Convar Setzen",
     "setconvarguide": "~r~~h~ACHTUNG~h~~w~: Wenn du nicht weißt was hier zu tun ist, nicht anfassen!~w~\nÄndert eine Convar.",
     "adminchangedconvar": "**%s** hat die Convar **%s** zu **%s** geändert.",
-    "adminstartedresource": "**%s** hat die Resource **%s** gestartet",
-    "adminstoppedresource": "**%s** hat die Resource **%s** gestoppt",
```
Comment (Blumlaut) [inline language/de.json:158]: Resource =/= Ressource

File: language/de.json [hunk 3]
```diff
@@ -214,13 +214,13 @@
...
-    "notification":"Benachrichtigung",
-    "deferral":"Prüfe Banliste, bitte warten.. (%s%%)",
-    "savebanchanges":"Save Changes",
-    "savebanguide":"~r~~h~NOTE:~h~~w~ Pressing Confirm will update this Ban on the Server."
+    "claimreport": "Report übernehmen",
+    "claimedby": "Übernommen von",
+    "reportalreadyclaimed": "Dieser Report wurde bereits übernommen",
+    "notification": "Benachrichtigung",
+    "deferral": "Prüfe Bannliste, bitte warten... (%s%%)",
+    "savebanchanges": "Speichere Änderungen",
```
Comment (Blumlaut) [inline language/de.json:222]: "Änderungen Speichern" would fit better, no?

File: language/de.json [hunk 4]
```diff
@@ -214,13 +214,13 @@
...
+    "savebanguide": "~r~~h~ACHTUNG:~h~~w~ Durch das drücken von bestätigen wird der Bann überarbeitet"
```
Comment (Blumlaut) [inline language/de.json:223]: Überschrieben, it's not getting "reworked"

**Review by Blumlaut — CHANGES_REQUESTED** [review]

**Comment by zRxnx** [comment]
should be fixed

File: language/de.json [hunk 5]
```diff
@@ -161,7 +161,7 @@
     "convarvalue": "Convar Inhalt",
 
     "cleanarea": "Gebiete aufräumen",
-    "cleanareaguide": "Entfernt alle Objekte in der nähe.",
+    "cleanareaguide": "Entfernt alle Gegestände in der nähe.",
```
Comment (Blumlaut) [inline language/de.json:164]: incorrect spelling

**Comment by zRxnx** [comment]
fixed

**Pull request merged**